server/survey/collectImport/jobs/collectIdmlParseUtils.js

```diff
@@ -0,0 +1,40 @@
+const R = require('ramda')
```

Comment:
I would call this file collectIdmlParser.

Reply:
in that case it should be a class, not just a set of helper methods

Reply:
no but why? just for three methods?